docker/build_scripts/build-git.sh

```diff
@@ -7,13 +7,13 @@ set -exuo pipefail
 # Get script directory
 MY_DIR=$(dirname "${BASH_SOURCE[0]}")
 
-if [ "${POLICY}" == "musllinux_1_1" ]; then
-	export NO_REGEX=NeedsStartEnd
-fi
-
 # Get build utilities
 source $MY_DIR/build_utils.sh
 
+if [ "${BASE_POLICY}" == "musllinux" ]; then
```

Comment:
Moved to get access to BASE_POLICY from utils

Reply:
llr.76